test/permessage-deflate.test.js

```diff
@@ -390,6 +390,55 @@ describe('PerMessageDeflate', function () {
       });
     });
 
+    it('honors the `zlibOptions` option', function (done) {
+      const lev9 = new PerMessageDeflate({ threshold: 0, zlibOptions: {level: 9, chunkSize: 128} });
```

Comment:
Can you please keep the limit of 80 chars, if possible?

Reply:
There's tons of this file that goes out to 100 characters, does it really matter?

Reply:
I know, my idea is to add prettier sooner or later so feel free to ignore this comment.